xdsl/backend/csl/lowering/convert_stencil_to_csl.py

```diff
@@ -0,0 +1,399 @@
+from attr import dataclass
+
+from xdsl.dialects import func
+from xdsl.dialects.arith import Cmpi, Constant, MinUI, Muli, OrI
```

Comment:
Instead of importing individual operations, we usually import the module, and then access operations as `arith.Muli`. This makes the resulting code much more readable, as you can immediately tell to which dialect an operation belongs.

Reply:
What do you mean by main dialect? But usually this is done for all dialects.

Reply:
I mean csl, sorry